Code-the-Dream-School/MyKitchenApp-front
# Fix signin redirect dev

## Description

Fix sign-in redirection to dashboard
## Resouce links or research articles


## Related Issue
closes #119 


## Acceptance Criteria
Go to the dashboard after login


## Type of Changes



|     | Type                       |
| --- | -------------------------- |
|     ✓| :bug: Bug fix              |
|   | :sparkles: New feature     |
|    | :hammer: Refactoring       |
|    | :100: Add tests            |
|    | :link: Update dependencies |
|    | :scroll: Docs              |

## Updates

### Before




### After



## Testing Steps / QA Criteria

Before logging in

- Try to go to private routes like `/dashboard`, `/profile` etc.
- Make sure they are not accessible and redirected to landing page

After logging in

- After clicking sign in, observe user is redirected to `/dashboard`
- Observe correct user name is displayed
